### ISSUE TITLE
Allow users to hold zoom level so that following pages are displayed similarly / legibly

### ISSUE THREAD

**Issue by comstock** [IIIF/mirador#742]
Using Mirador for reading requires the user first zoom to where the text can be read comfortably by the user. For most digitized books and manuscript collections, the zoom level that works for the current page-image is going to be a good zoom level for the next page, and the page after...

Once a user finds the zoom level that works, offer to (or make it the default behavior) to present all following pages at the same zoom level -- until the user changes the zoom and then hold that new level for subsequent page-views.

Manuscript example: http://iiif.lib.harvard.edu/manifests/view/drs:50257681$3i

**Comment by marlo-longley** [comment]
This directly conflicts with the user ask in #4217 
We should probably make this a configurable setting. I believe preserveViewport should cover this, but verify.

**Comment by andrewjbtw** [comment]
If the user is given the choice to hold the zoom across pages, I don't think this request necessarily conflicts with #4217 . 

Speaking as someone who has read a number of books in other image display interfaces - HathiTrust, publisher interfaces, etc. - and who is nearsighted enough to pretty much always need to zoom in on a single-page view, having to re-zoom every time I turn a page is basically a dealbreaker for reading a whole book. I've bounced off of reading books in Mirador for this and additional reasons (no keyboard arrow navigation) in the past. 

I understand the use cases for resetting the zoom each image and don't have a strong opinion on what the default should be. I don't think any default option could satisfy all use cases for the variety of materials Mirador presents, which makes me favor a user-configurable option if that's possible.

**Comment by dnoneill** [comment]
This is in conflict with https://github.com/ProjectMirador/mirador/issues/4217. I think maybe this should be a user setting?

**Comment by marlo-longley** [comment]
@dnoneill isn't this `preserveViewport`?
We set the default to false in https://github.com/ProjectMirador/mirador/pull/4220, but it should be possible to set to true and that would hold the zoom level.

**Comment by dnoneill** [comment]
@marlo-longley Yes this is. and yes we would be able to set it to true but I think the reason for #4220 is it is confusing for images. Also it can make things kinds of weird when you have different sized items. I think this seems either like we determine if it is a book thing or a user setting where a user can set their zoom level and then it would re-size to the correct size for the next page. I think it is difficult to use preserveViewport when it could be annoying for differently sized pages. That is why I think having a user setting that is like set zoom to 1.5x or something like that is both a nice feature and what users might expect. I don't know I think there are two things pushing against each other.
